insitu/inc/filtered_view.hpp

```diff
@@ -122,6 +122,11 @@ public Q_SLOTS:
 
     void restore(const Json::Value& json);
 
+    // Access selected topic and image dimensions outside insitu core
+    static std::string topic_ref_;
```

Comment:
This will not work with more than one filtered view on different topics. See this resource [on static member variables](https://www.learncpp.com/cpp-tutorial/static-member-variables/) in C++.

Reply:
This works just fine for what I tested. 
I need a way to interact with the root image. As it stands, filters have no bearing to the image they are put, which is good. However, if a filter needs to use something in the image its being put on, this isn't provided. 
If you come round to it, please provide a way t do this in a filter. Hopefully soon.

Reply:
Thanks. Will do.